portal/gatsby-config.js

```diff
@@ -6,5 +6,36 @@ module.exports = {
         description: `Documentation of Jøkul design system`,
         author: `Fremtind`,
     },
-    plugins: ["gatsby-plugin-typescript", "gatsby-plugin-sass"],
+    plugins: [
+        "gatsby-plugin-typescript",
+        {
+            resolve: `gatsby-plugin-sass`,
+            options: {
+                implementation: require("sass"),
```

Comment:
Bruker dart-sass isteden for node-sass, burde løse sikkerhetsproblemene vi har i dag, ref snyk.